src/ops/ops.go

```diff
@@ -463,3 +465,44 @@ func (o *ops) CreateOpenshiftSshManifest(filePath, tmpl, sshPubKeyPath string) e
 	}
 	return nil
 }
+
+func (o *ops) GetMustGatherLogs(workDir string, kubeconfigPath string) (string, error) {
+	//invoke oc adm must-gather command in the working directory
+	o.log.Infof("collecting must-gather logs into %s", workDir)
+	command := fmt.Sprintf("cd %s && oc --kubeconfig=%s adm must-gather", workDir, kubeconfigPath)
```

Comment:
@slaviered @tsorya u can use dest dir, no need to cd:
https://github.com/openshift/oc/blob/8ad72f131e0c55c93b9ceac7180f9096095f75a8/pkg/cli/admin/mustgather/mustgather.go#L60